src/graphql/Hero/hero-schema.graphql

```diff
@@ -26,7 +31,7 @@ type Mutation {
 }
 
 type Query {
-  heroes(type_id: ID, name_query: String, skip: Int, first: Int): [Hero!]!
+  heroes(type_id: ID, name_query: String, skip: Int, first: Int): HeroesResponse
```

Comment:
What do you thing about implementing pagination in more standard (for GraphQL) way - `https://graphql.org/learn/pagination/` ?

Reply:
@fan17 Actually, `skip` and `first` is the convention for handling pagination in Prisma, and for me, it's a pretty intuitive one. The Same approach we use for REST API and I'd prefer to stay with this one for now :)